tools/events-automation/event.py

```diff
@@ -1,3 +1,7 @@
+from datetime import datetime
+from geopy.geocoders import Nominatim
+from state_territory_to_abbrev import au_state_territory_to_abbrev, us_state_to_abbrev, ca_state_territory_to_abbrev
+from country_to_abbrev import country_to_abbrev
 
 class Event():
```

Comment:
With the current behavior in test_events.py, we need to add `organizerUrl` to the `Events` struct